pkg/model/keycloak_deployment.go

```diff
@@ -28,6 +28,22 @@ func GetServiceEnvVar(suffix string) string {
 	return fmt.Sprintf("%v_%v", serviceName, suffix)
 }
 
+func getPodSecurityContext(secctx *v1.PodSecurityContext) *v1.PodSecurityContext {
+	if secctx != nil {
+		return secctx
+	}
+
+	defaultUserAndGroup := int64(1000)
+	runAsNonRoot := true
+
+	return &v1.PodSecurityContext{
+		RunAsUser:    &defaultUserAndGroup,
+		RunAsGroup:   &defaultUserAndGroup,
+		RunAsNonRoot: &runAsNonRoot,
+		FSGroup:      &defaultUserAndGroup,
+	}
```

Comment:
This is potentionally a breaking change, we weren't setting any defaults before. Could you please remove it?